redisinsight/ui/src/styles/themes/light_theme/_theme_color.scss

```diff
@@ -156,7 +156,7 @@ $recommendationBorderColor: #3953c3;
 
 
 // Recommendations
-$recommendationsBgColor: #FFFFFF;
+$recommendationsBgColor: #F0F0F0;
```

Comment:
I noticed that this color is being used in multiple places in the codebase. Is it safe to change it here, globally?